tests/test_modelscan.py

```diff
@@ -456,7 +452,10 @@ def test_scan_zip(zip_file_path: Any) -> None:
     ms = ModelScan()
     results = ms.scan(f"{zip_file_path}/test.zip")
     assert results["summary"]["scanned"]["scanned_files"] == [f"test.zip:data.pkl"]
-    assert results["summary"]["skipped"]["skipped_files"] == []
+    assert [
+        skipped_file["source"]
+        for skipped_file in results["summary"]["skipped"]["skipped_files"]
+    ] == ["test.zip"]
```

Comment:
This might still be little confusing since test.zip was scanned for it's content.